### ISSUE TITLE
2022 Competition/Call for Entries Themes 

### ISSUE THREAD

**Issue by mayags** [Third-Coast/website#53]
Access
Equity
Community
Shared
Examine
Explore
Intention
Care
Consider 
Collectivity 
Collaboration 
Invitation 
Offerings
Exchange

**Comment by TCEmkenn** [comment]
We want to move away from: 

Institutional
Self-Serious
Remote

& move towards: 

Inviting
Clear
Transparent

**Comment by brendanmetzger** [comment]
@TCEmkenn @mayags 

Interesting notes. I've been playing with color this afternoon, and some typefaces. There are a few looks here, I wanted to keep the color decision-less for now, based off a CMYK palette. I set the typeface to reflect a tone that could carry h1s and some of the sidebar stuff as discussed. lemme take you through my thoughts—it feels like to get a stronger theme across without going down the design rabbit hole,  an emotive typeface can pull a lot of weight, so I wanted to show some examples..

------
this typeface has a lo-fi game vibe, which brings to mind collectivity, collaboration, community, inviting...
<img width="1552" alt="Screen Shot 2022-10-11 at 4 42 29 PM" src="https://user-images.githubusercontent.com/4213/195204863-5b005632-9374-471a-838d-c29c39bf2a68.png">

----

This typeface kinda has a cool origin.. it was designed for a google doodle celebrating the 40th(?) anniversary of the creation of hip-hop—it's look is based on the graffiti in brooklyn I think. Again, has a 'collective' vibe to me

<img width="1552" alt="Screen Shot 2022-10-11 at 4 41 24 PM" src="https://user-images.githubusercontent.com/4213/195204871-77dd6e82-7176-4d22-9cb8-8e25aaa5c554.png">

-----

monospace tried on for size. Without playing harder with what monospace can do in terms of proportions and layouts, it will feels institutional. This one is a tad craft-y to get around that for now.

<img width="1552" alt="Screen Shot 2022-10-11 at 4 41 05 PM" src="https://user-images.githubusercontent.com/4213/195204875-163fc644-4ec3-4565-8a14-4d70e1020f16.png">

----

this one is called marker felt. While handwritten can be over the top, I've always liked this one because it is consistent/precise and definitely has a "we are sketching this out" vibe, which inherently feels less institutional and more explore-y..

<img width="1552" alt="Screen Shot 2022-10-11 at 4 39 38 PM" src="https://user-images.githubusercontent.com/4213/195204880-c481a7a6-4e2c-4e30-b36f-7735830479e9.png">

**Comment by brendanmetzger** [comment]
<img width="1552" alt="Screen Shot 2022-10-12 at 7 41 58 PM" src="https://user-images.githubusercontent.com/4213/195473946-0c84f7ad-be82-4f64-a1c8-895636db849b.png">
<img width="1552" alt="Screen Shot 2022-10-12 at 7 40 06 PM" src="https://user-images.githubusercontent.com/4213/195473952-b89ae522-14bd-446e-8e8c-d59e7d0f6429.png">
<img width="1552" alt="Screen Shot 2022-10-12 at 7 37 14 PM" src="https://user-images.githubusercontent.com/4213/195473955-a4c2e219-2e43-44ba-9510-03ffb6ceaf31.png">
<img width="1552" alt="Screen Shot 2022-10-12 at 7 33 53 PM" src="https://user-images.githubusercontent.com/4213/195473957-071ad4d0-a29a-4695-8693-3982fe6e60ac.png">
<img width="1552" alt="Screen Shot 2022-10-12 at 7 30 09 PM" src="https://user-images.githubusercontent.com/4213/195473961-19b712ae-e73b-4ad1-9ebf-fb6f261fd809.png">
<img width="1552" alt="Screen Shot 2022-10-12 at 7 21 46 PM" src="https://user-images.githubusercontent.com/4213/195473963-fe8fd936-c23a-4d8b-a232-6062101e2772.png">
<img width="1552" alt="Screen Shot 2022-10-12 at 7 14 36 PM" src="https://user-images.githubusercontent.com/4213/195473964-0545b8af-a2d5-43b5-8c75-b6e73bb6fcd9.png">

**Comment by mayags** [comment]
Hey there: thank you for these, Brendan! I like Futura condensed, and also Unit Rounded. Would it be possible to see these -- maybe next week -- using different colors (perhaps from our color template) as well? Thank you!

**Comment by TCEmkenn** [comment]
Hi there — ooo, fun! I like Futura Condensed, also have always loved Neue Haas Grotesk?

Can I see Neue Haas Grotesk in a slightly less-heavy weight? (Like a semi-bold?)

**Comment by TCEmkenn** [comment]
These are so fun! I like Futura Condensed, and have always love Neue Haas Grotesk. Can we take down the weight of Neue Haas Grotesk + see what that looks like? (Is there a semibold?)

**Comment by mayags** [comment]
Would also love to see Neue Haas Grotesk in a semi-bold! 

In terms of color/palette: what do y'all think of continuing to use & play with colors from the banner on the homepage?

![Pidgey Header](https://user-images.githubusercontent.com/8697911/196736117-9ccd1e4b-9187-4f88-ad42-14dfa5397906.png)

**Comment by brendanmetzger** [comment]
in order:

- [x] dark blue for banner
- [x] classic enter button is green/ready as existing
- [x] made the extra/billboard #d69c8a
- [ ] Avenir is not available on typekit to embed consistently. It is standard on macs (it is the system default on iPhones) but that will not be consistent across devices without an embed

**Comment by brendanmetzger** [comment]
> One small change/question: can the button "Apply Now" read "Enter Now"? Also, can appear across all pages related to the Competition? Is it possible for us to decide where to place that button on the sidebar, or is it set?

You can set the language, and it can be applied on any page individually or across a series of connected pages. Eg, if you put a button on the categories page, then all the individual categories connected will have an apply button too.

the language you need to execute is `- [Enter Now](link to form)`

**Comment by mayags** [comment]
Ok, thanks!

I just tried that out here for the button on this page: https://thirdcoastfestival.org/explore/detail/BwZ

What am I getting wrong?

**Comment by brendanmetzger** [comment]
@mayags it has to go in the 'extra/billboard' section

**Comment by mayags** [comment]
> @mayags it has to go in the 'extra/billboard' section

Hmm, I've tried that on the category page: - [Enter Now](https://https2022competition.paperform.co/) is in the Extra Section.

**Comment by brendanmetzger** [comment]
@mayags oh, sorry, none of this will show up yet until I push my code. It will look like what I have in the screenshot though

**Comment by mayags** [comment]
Hi Brendan:

I think the "Category" pages are looking better and better: I like the color for the banner & the billboard. 

I just edited the "billboard" on the Best Documentary page -- I think it will be closer to this length, rather than the longer copy we were playing with before. With that in mind, could the "On this Page" section live just below the billboard?

Would it be possible to make the following pages consistent with the "category" pages (in terms of the banner + sidebar + billboard?): [Landing Page](https://www.thirdcoastfestival.org/competition/2022-third-coastrichard-h-driehaus-foundation-competition), the "roadmap" pages aka the [Categories Page](https://www.thirdcoastfestival.org/article/competition-categories), [Offerings Page](https://www.thirdcoastfestival.org/explore/detail/Bxu), [Entry Guide Page,](https://www.thirdcoastfestival.org/explore/detail/XG) and [Entry Rates Breakdown](https://www.thirdcoastfestival.org/explore/detail/Bwo), plus our "Vision + Values" pages -- the [Looking to the Future Page](https://www.thirdcoastfestival.org/explore/detail/Bxx), [History of the Competition Page ](https://www.thirdcoastfestival.org/explore/detail/BvH) and [Collective Pricing Model Page](https://www.thirdcoastfestival.org/explore/detail/Bwd) (We may add on more page in this category!)

**Comment by brendanmetzger** [comment]
@mayags I've updated the sidebar on all the pages to have the gradient. Unfortunately the banner won't have an immediate solution, as those pages are technically still an article template, unlike the category page. I'll ponder it though and see if there is anything I can come up with that doesn't involve creating a new template.

That said, I have v. limited time left today, so it might not be something to wait on!

**Comment by mayags** [comment]
@brendanmetzger A few very specific questions/changes for you: 

1) On the category pages, the H6 function (which allows the text to be clicked, and then more text to drop down) is no longer working! We really like/want the functionality of that -- we want everything below "How to Enter: Best Documentary" (or "How to Enter: X Category" on whatever category page) to mirror the way that each individual offering function on the [Offerings Page](https://www.thirdcoastfestival.org/explore/detail/Bxu). Does that make sense & can you fix that?

2) We're liking the way the "billboard" is looking more and more, for example on the [Best Documentary](https://www.thirdcoastfestival.org/collection/best-documentary-bd) page. We want to keep that copy inside the orange-y gradient "billboard" box, but DELETE the old copy from the old "billboard" or Extra section. That's the copy underneath the "Enter Now" button until the "Related Pages" section. How do we (or you) delete that copy?

3) The highlight function for "On this Page" is a really cool accessibility tool. Can the highlight color please be a different color than the bright yellow? Could it perhaps match the billboard color, or even the green?

4) On the [Competition Categories](https://www.thirdcoastfestival.org/explore/detail/BwZ) page, why is the "Enter" button inside of the billboard? Is it possible for the Enter button to live below the billboard, like on the other pages?

**Comment by mayags** [comment]
Two more urgent things @brendanmetzger : 

- We are missing the Best Documentary: Short page, which I'd realized before but naively thought we could recreate. However, since you've developed this new template, I don't think that's true. Brendan: Do you know where this page went? Alternatively, if we re-create the Best Documentary: Short page as an article, would you be able to change it into the "category page" template?

- We have a new URL for the Competition Entry Form due to an error from our formbuilder (who we love!) and so we need to replace the URL on all of the "Enter Now" buttons that you created and that we don't have access to. Here is the new URL: https://2022thirdcoastcompetition.paperform.co/

**Comment by brendanmetzger** [comment]
I only have a minute at the moment, but change the form url on the category page extra I think

In order to make a new award, it needs to be a collection/playlist, then attach it to the main category page and you'll be good

**Comment by mayags** [comment]
Hey -- so I am able to insert an Enter Now button the category page, but there's one Enter button that we don't have access to/can't edit... right? See here: https://www.thirdcoastfestival.org/explore/detail/Bwh

**Comment by mayags** [comment]
Hi there:

I see some of the issues above got resolved, yay!

Still wondering about the following:

We're liking the way the "billboard" is looking more and more, for example on the [Best Documentary](https://www.thirdcoastfestival.org/collection/best-documentary-bd) page. We want to keep that copy inside the orange-y gradient "billboard" box, but DELETE the old copy from the old "billboard" or Extra section. That's the copy underneath the "Enter Now" button until the "Related Pages" section. How do we (or you) delete that copy?

Also: is possible to include the "Enter Now" button on every page, consistent with how it's set up on the "category" pages? (In other words, as a separate "Enter Now" button, rather than inside of the "Billboard")

**Comment by brendanmetzger** [comment]
@mayags 

> We're liking the way the "billboard" is looking more and more, for example on the [Best Documentary](https://www.thirdcoastfestival.org/collection/best-documentary-bd) page. We want to keep that copy inside the orange-y gradient "billboard" box, but DELETE the old copy from the old "billboard" or Extra section. That's the copy underneath the "Enter Now" button until the "Related Pages" section. How do we (or you) delete that copy?

To delete the copy, you would remove it from the [competition categories](https://www.thirdcoastfestival.org/article/competition-categories) page. That parent page's 'billboard' is appended to each connected category page, which has the effect of putting the enter now on every page, which leads to your second ?...

> Also: is possible to include the "Enter Now" button on every page, consistent with how it's set up on the "category" pages? (In other words, as a separate "Enter Now" button, rather than inside of the "Billboard")

As long as the link is in the extra/billboard section, it will be applied to all category pages (along with whatever copy you spec). There is now a caveat I hadn't accounted for when you asked to make the article pages have the orange-gradient 'billboard', which I'll try and articulate cleanly..

- every page can have a billboard
- the billboard of each page will be in an orange gradient
- 'child pages' (ie, Best Documentary) embed parent pages billboard (ie., categories page)
- parent billboards are not orange when embedded in child pages

The rationale was to provide consistent language across child pages as the competition information needs to be updated, rather than updating lots of individual pages manually.

I can simply take away the orange box on article pages and make plain, or make 'parent' pages no longer share their billboard with 'child' pages, which removes the button from every page and would have to be added manually to each category (though then it would be in an orange box!)

Lastly, I can hard code it into the template to step outside of the orange box for the button, but should anything change, I would have to recode the template, which given the turbulence of my workload lately, may not be ideal  😕

**Comment by mayags** [comment]
Hi hi:

Ok, I think (?) I am understanding. 

- Would it be possible to make parent pages no longer share their billboard with child pages, so we could do that manually?

- I think you can go ahead and hard code the "Enter Now" button to live outside the orange box, and on every page related to the Competition (the index of those pages is below) -- because the URL for the entry form won't change. Is there something else you're worried about changing that could disrupt this? 

Also, wanted to bring your attention to this point from one of the posts above, which I think is also still unresolved: The highlight function for "On this Page" is a really cool accessibility tool. Can the highlight color please be a different color than the bright yellow? Could it perhaps match the billboard color, or even the green?

Index of Pages to include an "Enter Now" button outside of the orange "billboard":

ROADMAP-PAGES:
Competition Categories 
[2022-23 Offerings](https://thirdcoastfestival.org/explore/detail/Bxu)
[Entry Guide](https://thirdcoastfestival.org/explore/detail/XG) 

NUTS & BOLTS
[Entry Rates Breakdown](https://www.thirdcoastfestival.org/explore/detail/Bwo) 
[General FAQ](https://www.thirdcoastfestival.org/explore/detail/BvK)

CATEGORY PAGES: 
Best Documentary 
Audio Unbound 
[Best Documentary: Non-English Language](https://www.thirdcoastfestival.org/explore/detail/Bwk) 
Best Documentary: Short 
[Best New Artist](https://www.thirdcoastfestival.org/explore/detail/Bwj)
[Best News Feature](https://www.thirdcoastfestival.org/explore/detail/Bwn) 
[Impact](https://www.thirdcoastfestival.org/explore/detail/Bwb)
Best Serialized Story 
[Directors’ Choice](https://www.thirdcoastfestival.org/explore/detail/Bxh)

VISION & VALUES PAGES
Looking to the Future 
More on our Process 
[Collective Pricing Overview](https://www.thirdcoastfestival.org/explore/detail/Bwd)
[History of the Competition](https://www.thirdcoastfestival.org/explore/detail/BvH)

**Comment by brendanmetzger** [comment]
-FYI — If I hardcode a button into a template it will show have to show up everywhere, not just that list of article pages. The category pages have their own template that I made, so that page can be hardcoded in. The only solution for the remaining pages is to manually put the 'apply button' in the extra section, then remove the orange gradient from article pages to address the aesthetic concern. (the thing I am trying to import now though, is if that's what we ultimately do, then that's essentially what is in place now).
- the highlight thing is already done

**Comment by brendanmetzger** [comment]
I have a little more time to update what I can—I still need to change the competition title, which should only take a few minutes, but I'd like to package the last set of changes together

**Comment by mayags** [comment]
Hi @brendanmetzger 

I think I am good with you hardcoding the "Enter Now" button so that it appears on all pages, as long as it's something that we can remove once the Competition closes on March 10. 

If the "Enter Now" button lived somewhere else on the page, could it be placed on the index of pages above, rather than every page on the website?

**Comment by brendanmetzger** [comment]
> If the "Enter Now" button lived somewhere else on the page, could it be placed on the index of pages above, rather than every page on the website?

I'm not sure I follow the placement you refer to. Generally, text that says 'enter now' without any context is a bit hard for me to get behind, but I can certainly do it.

**Comment by mayags** [comment]
Ok -- scratch that thought. I am not totally sure if I'm following.

**Comment by mayags** [comment]
Ok, a point of clarification, in case this helps:

What I think we want is for an "Enter Now" button to only appear on the constellation of Competition pages that we're introducing today -- more on that below.

The exact placement of the "Enter Now" button that we're looking for would be on the sidebar, below the orange "billboard" and above the "On this page" menu. 

The constellation of Competition pages we're hoping the "Enter Now" button would appear on is listed below, as an index:

ROADMAP-PAGES:
[Competition Categories](https://www.thirdcoastfestival.org/explore/detail/BwZ)
[2022-23 Offerings](https://thirdcoastfestival.org/explore/detail/Bxu)
[Entry Guide](https://thirdcoastfestival.org/explore/detail/XG)

NUTS & BOLTS
[Entry Rates Breakdown](https://www.thirdcoastfestival.org/explore/detail/Bwo)
[General FAQ](https://www.thirdcoastfestival.org/explore/detail/BvK)

CATEGORY PAGES:
[Best Documentary](https://www.thirdcoastfestival.org/explore/detail/Bwh)
[Audio Unbound](https://www.thirdcoastfestival.org/explore/detail/Bwm)
[Best Documentary: Non-English Language](https://www.thirdcoastfestival.org/explore/detail/Bwk)
[Best Documentary: Short](https://www.thirdcoastfestival.org/explore/detail/ByB)
[Best New Artist](https://www.thirdcoastfestival.org/explore/detail/Bwj)
[Best News Feature](https://www.thirdcoastfestival.org/explore/detail/Bwn)
[Impact](https://www.thirdcoastfestival.org/explore/detail/Bwb)
[Best Serialized Story](https://www.thirdcoastfestival.org/explore/detail/BwW)
[Directors’ Choice](https://www.thirdcoastfestival.org/explore/detail/Bxh)

VISION & VALUES PAGES
[Looking to the Future](https://www.thirdcoastfestival.org/explore/detail/Bxx)
[More on our Process](https://www.thirdcoastfestival.org/explore/detail/ByC)
[Collective Pricing Overview](https://www.thirdcoastfestival.org/explore/detail/Bwd)
[History of the Competition](https://www.thirdcoastfestival.org/explore/detail/BvH)

**Comment by brendanmetzger** [comment]
Right, so all of these are article templates except for the 'category pages', so if I hardcode into the template, the screenshot below will happen. If you want the enter button *only* on that list, it has to be added to the 'extra/billboard' portion of that page, but it will have the orange background (unless I remove the orange background from article pages).

<img width="1237" alt="Screen Shot 2022-10-28 at 12 00 01 PM" src="https://user-images.githubusercontent.com/4213/198692275-6875688d-8fea-409f-828a-feee1f82b3c4.png">

**Comment by mayags** [comment]
Got it. 

Ok, @brendanmetzger, for now: let's not hard code the "Enter Now" button into the article template. You can leave the "Enter Now" button as you've set it up on the "category" pages. Also for now, let's leave the orange billboard up (and we'll include the "Enter Now" button in the billboards on those pages.)
